datasketch/lsh.py

```diff
@@ -282,6 +301,39 @@ def _insert(
         for H, hashtable in zip(Hs, self.hashtables):
             hashtable.insert(H, key, buffer=buffer)
 
+    def __equivalent(self, other:MinHashLSH) -> bool:
+        """
+        Returns:
+            bool: If the two MinHashLSH have equal num_perm, number of bands, size of each band and hashfunc (if provided) then two are equivalent.
+        """
+        return (
+            type(self) is type(other) and
+            self.h == other.h and
+            self.b == other.b and
+            self.r == other.r and
+            type(self.hashfunc) == type(other.hashfunc)
+        )
+
+    def _merge(
+        self,
+        other: MinHashLSH,
+        check_disjointness: bool = False,
```

Comment:
```suggestion
        check_disjoint: bool = False,
```

Reply:
The argument is changed to check_overlap